op-chain-ops/crossdomain/precheck.go

```diff
@@ -67,7 +77,13 @@ func PreCheckWithdrawals(db *state.StateDB, withdrawals []*LegacyWithdrawal) ([]
 			continue
 		}
 
-		filtered = append(filtered, slotsInp[slot])
+		wd := slotsInp[slot]
+		if wd.Sender == nil || *wd.Sender != predeploys.L2CrossDomainMessengerAddr {
```

Comment:
Should this be `&&`?

Reply:
If we use `&&` this will panic - we want to exit if sender is null or it isn't the `L2XDM`.